scripts/packaging/createupmpackages.ps1

```diff
@@ -31,16 +31,15 @@ if (-not $ProjectRoot) {
     throw "Missing required parameter: -ProjectRoot."
 }
 $ProjectRoot = Resolve-Path -Path $ProjectRoot
-Write-Output "Project root: $ProjectRoot"
 
 if (-not $Version) {
     throw "Missing required parameter: -Version."
 }
 
-if (-not $IsOfficial) {
-    if (-not $BuildNumber) {
-        throw "Missing required parameter: -BuildNumber. This parameter is required when -IsOfficial is set to false."
-    }
+if ((-not $BuildNumber) -and $useBuildNumber) {

```

Comment:
$UseBuildNumber  instead of $useBuildNumber (here and below)

PS is case insensitive in variable naming but that's super weird.

Reply:
fixed